# PWM signals for the three robots of a team
RobotPWM[3] robot_pwm